src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.links.strings

```diff
@@ -10,6 +10,8 @@
 
 "safari_corona_social_media" = "https://www.coronawarn.app/en/community/#socialmedia";
 
+"safari_corona_accessibility" = "https://www.coronawarn.app/en/accessability";
```

Comment:
```suggestion
"safari_corona_accessibility" = "https://www.coronawarn.app/en/accessibility";
```
Also to be corrected for other languages where EN link is used

Reply:
@KathrinLuecke I updated it, The typo was in the jira ticket.